observer/test/observer.test.js

```diff
@@ -89,7 +124,7 @@ describe('observer', () => {
       ieContractMock.queryFilter = async (eventName, fromBlock) => {
         const events = [
           { args: { to: 'address1', amount: 50 }, blockNumber: 2000 },
-          { args: { to: 'address1', amount: 50 }, blockNumber: 2000 }
+          { args: { to: 'address2', amount: 150 }, blockNumber: 2000 }
```

Comment:
We need to revert this change, as it significantly changes what is being tested in this test.